src/cockpit/packages.py

```diff
@@ -192,11 +193,18 @@ def relative_to(self, path):
         assert x.startswith(y)
         return Path(x[len(y):])
 
+# TODO: This doesn't yet implement the slighty complicated checksum
```

Comment:
This *does* implement the checksum scheme, and even does it in exactly the same way (which was needed to get unit tests passing).  But reloading probably does need more work, indeed.